chess/frontend/gui/src/modules/Dashboard/subcomponents/LeftSidebar.jsx

```diff
@@ -118,6 +130,19 @@ export class LeftSidebar extends React.Component {
           )
         }
 
+        {
+          this.state.opponent === 'Online Opponent' && (
```

Comment:
Is this a conditional check and a statement? I've never seen anything like this before